spec/rubocop/config_loader_spec.rb

```diff
@@ -386,6 +387,14 @@
       )
     end
 
+    it 'loads configuration properly when it includes non-ascii characters ' do
+      create_file(configuration_path, ['# All these cops of mine are ❤',
```

Comment:
Sorry, i realized I should have included the command in my previous comment after posting it. Its there now.